src/urls.ts

```diff
@@ -0,0 +1,57 @@
+import * as fs from 'fs';
+import * as jsdom from 'jsdom';
+import * as path from 'path';
+import syncRequest = require('sync-request');
+
+// tslint:disable-next-line:no-var-requires
+const TurndownService = require('turndown');
+
+const pathToCache = path.join(__dirname, 'data/urls.json');
+
+// tslint:disable-next-line:no-var-requires
+const urlData: Record<string, string> = require(pathToCache);
+
+const turndownService = new TurndownService();
+
+// change anchors to plain text so we don't end up with a bunch of relative urls
+turndownService.addRule('anchor', {
+  filter: 'a',
+  replacement: (content: string) => content,
+});
+
+function scrapeSummary(url: string): string {
+  try {
+    const htmlContents = syncRequest.default('GET', url).getBody() as string;
+    const result = new jsdom.JSDOM(htmlContents);
+    const summaryElement = result.window.document.querySelector('#wikiArticle > p:not(:empty)');
+    if (summaryElement) {
+      return turndownService.turndown(summaryElement.innerHTML);
+    }
+    return '';
+  } catch (ex) {
+    return '(Description not available)';
```

Comment:
Do we want this or should the string be empty and not generated into the JSDOC at all? Because properties that's missing compatibility data or the `mdn_url` will be ignored too.

A warning in the terminal would be suitable though (as for when the `summaryElement` is missing) to detect changes on their website since this solution is fairly fragile.

Reply:
If we leave it empty, it will try to reach out during each build. I guess that would make it easier to automatically fix these as they get fixed on mdn. 

If you are cool with that, an empty string would work fine here.  I'll a console warning as well.

Reply:
That's fine due to the fact that the link is wrong and should be fixed at MDN.

Reply:
Created an issue https://github.com/mdn/browser-compat-data/issues/1998